### PR TITLE
Cleanup apps

### PR DESCRIPTION
**Pull request by bkeepers**
- Removes apps without docs
- Limits homepage to 6 apps
- Syncs stars

Next I'll extract `/apps` from #18 so there is a listing of all apps.

### CODE REVIEW

File: _apps/stale.md [hunk 1]
```diff
@@ -8,9 +8,8 @@ screenshots:
 topics:
 - project-management
 author: bkeepers
-stars: 129
+stars: 128
```
Comment (hiimbex) [inline _apps/stale.md]: 🤣

Comment (bkeepers) [inline _apps/stale.md]: 😧

Comment (anglinb) [inline _apps/stale.md]: oh shit

**Pull request merged**